web/styles/message_row.css

```diff
@@ -203,7 +203,8 @@
             */
             padding: 5px 0 0;
             color: var(--color-text-message-default);
-            line-height: var(--message-box-line-height);
+            font-size: var(--base-font-size-px);
```

Comment:
The goal is to keep moving the element on which these values are set ever further up the DOM tree. That will allow work to include ever-larger swaths of the UI, with adjustments to dependent values as things progress.